File: CPP/rad_2_deg.cpp
#include <iostream>
#include <cmath>

int main() {
    // Declare variables to store radians and degrees
    double radians, degrees;

    // Prompt the user for input
    std::cout << "Enter radians: ";
    std::cin >> radians;

    // Convert radians to degrees using the formula
    degrees = radians * (180.0 / M_PI);

    // Display the result
    std::cout << "Degrees: " << degrees << std::endl;

    return 0;
}

File: Python/rad_2_deg.py
import math

# Function to convert radians to degrees
def radians_to_degrees(radians):
    degrees = radians * (180.0 / math.pi)
    return degrees

# Input the value in radians
radians = float(input("Enter radians: "))

# Convert radians to degrees using the function
degrees = radians_to_degrees(radians)

# Display the result
print("Degrees:", degrees)
